src/distributions/mod.rs

```diff
@@ -54,6 +54,7 @@ pub trait IndependentSample<Support>: Sample<Support> {
 
 /// A wrapper for generating types that implement `Rand` via the
 /// `Sample` & `IndependentSample` traits.
+#[derive(Copy)]
```

Comment:
This is more general if it is written explicitly `impl<Sup> Copy for RandSample<Sup>`, as the `#[derive]` will be placing a `Sup: Copy` bound in the impl, which isn't necessary in this case.